com.unity.shadergraph/CHANGELOG.md

```diff
@@ -121,8 +130,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Updated legacy COLOR output semantic to SV_Target in pixel shader for compatibility with DXC.
 - Updated the functions in the `Normal From Height` node to avoid NaN outputs.
 - Changed the Voronoi Node algorithm to increase the useful range of the input values and to always use float values internally to avoid clipping.
-- Changed the `Reference Suffix` of Keyword Enum entries so that you cannot edit them, which ensures that material keywords compile properly. 
-- Updated the dependent version of `Searcher` to 4.2.0. 
+- Changed the `Reference Suffix` of Keyword Enum entries so that you cannot edit them, which ensures that material keywords compile properly.
```

Comment:
What are all of these changes to the changelog?  Did your editor clean up newlines?

Reply:
Yup.  It's the same thing the conversion will do.

Reply:
(they were in the master PR as well)

Reply:
Yeah I saw.   Oh well  :)   Looks good!